prediction/tournament_output.py

```diff
@@ -0,0 +1,22 @@
+class TournamentOutput(object):
+    """An abstract data that represents the Tournament output of some model."""
+    def get_winner_prob(self, outcome):
+        pass
+
+    def get_nonzero_prob_outcomes_list(self):
+        pass
+
+class ConcreteTournamentOutput(TournamentOutput):
+
+    def __init__(self, tournament):
+        self.tournament = tournament
+    
+    def get_winner_prob(self,outcome):
+        outcome = outcome.lower()
+        if outcome in self.tournament:
+            return self.tournament[outcome]
+        else:
+            return 0
+
+    def get_nonzero_prob_outcomes_list(self):
```

Comment:
I would guess that in each tournament output, each country has its probability of making it to the round of 16, round of 8, semi final, final, and winning World Cup and each country will surely have non zero probabilities. So why is there this method?

Reply:
It was there incase input is a country not in round 16. (Those country will have zero probability)